docs/index.html

```diff
@@ -225,10 +225,12 @@ <h1 id="api-testing">Testing Aids</h1>
 echo Carbon::parse();                                  // 2001-05-21 12:00:00
 echo new Carbon('now');                                // 2001-05-21 12:00:00
 echo Carbon::parse('now');                             // 2001-05-21 12:00:00
-var_dump(Carbon::hasTestNow());                        // bool(true)
+var_dump(Carbon::hasTestNow());                        // C:\Users\bastien.miclo\Perso\Carbon\generate.php(44) : eval()'d code:1:
```

Comment:
`C:\Users\bastien.miclo\Perso\Carbon\generate.php(44)` should that be here?

Reply:
Shit, xdebug must be disabled for the script.